src/platform/packages/shared/kbn-es-query/src/filters/helpers/convert_range_filter.ts

```diff
@@ -12,20 +12,26 @@ import { keys } from 'lodash';
 import type { RangeFilter } from '../build_filters';
 import type { TimeRange } from './types';
 
+const isRelativeTime = (value: string | number | undefined): boolean => {
+  return typeof value === 'string' && value.includes('now');
```

Comment:
Not all relative time ranges include `now`. One of examples: `2001.02.01\|\|+1M/d` per https://www.elastic.co/guide/en/elasticsearch/reference/8.17/common-options.html#date-math

Reply:
The relative time in that case comes from our time picker and not from ES

Reply:
So I took it under consideration here https://github.com/elastic/kibana/pull/207046 but you need to check the other usages too. Also it would be good to extract all these helpers from the data plugin to a package for easier consumption